test/message/core_line_numbers.out

```diff
@@ -1,9 +1,9 @@
-punycode.js:42
+punycode.js:*
 	throw new RangeError(errors[type]);
 	^
 
 RangeError: Invalid input
-    at error (punycode.js:42:*)
```

Comment:
The whole test is only about this line number. Therefore I would rather not touch this / find a different solution for this.

Reply:
any recommendations? this test will never work on the transpiled code. One thought, perhaps perhaps extend on the matcher to support something like this:

` at error (punycode.js:(42|929):*)`

Reply:
That does not seem to be a reliable way as the line number could likely also change again without anyone noticing it and it could also produce falsy expected entries in other output files.

What might work is to pass a flag / option to the test runner and then the test runner just ignores all numbers when receiving patterns that look like ` +at .*.js:(?:\d+|\*)(?::(?:\d+|\*))?`.

That way this would never be an option anymore, also if new tests add line numbers and or columns.

Reply:
what if we just update the assertion to look at `\d+`; seems like the `42` is fragile ... and asserting that we have _a line number_ would be sufficient for this test.

Reply:
That does not work either. The test is about the specific line number.

Reply:
See https://github.com/bcoe/node-1/commit/b799a74709af69daf13901390df9428c4c38adfc#diff-fac9894e053cf4e578875b1166718a58 for more context.

Reply:
It seems like overkill to add a flag for one test; I'd be more tempted to add it to a skip list for coverage -- seems like a reasonable compromise.